packages/db/src/schema.prisma

```diff
@@ -164,6 +166,8 @@ model Word {
   translatorNotes TranslatorNote[]
   // The list of footnotes in different languages for this word.
   footnotes       Footnote[]
+  /// The gloss updates applied to this word.
+  glossHistory    GlossHistory[]
```

Comment:
I would move this up above `machineGlosses`

Reply:
Done.